src/transformers/configuration_utils.py

```diff
@@ -94,6 +91,16 @@ def __init__(self, **kwargs):
         self.label2id = kwargs.pop("label2id", dict(zip(self.id2label.values(), self.id2label.keys())))
         self.label2id = dict((key, int(value)) for key, value in self.label2id.items())
 
+        # Tokenizer arguments TODO: eventually tokenizer and models should share the same config
+        self.prefix = kwargs.pop("prefix", None)
+        self.bos_token_id = kwargs.pop("bos_token_id", None)
+        self.pad_token_id = kwargs.pop("pad_token_id", None)
+        self.eos_token_id = kwargs.pop("eos_token_id", None)
+        self.decoder_start_token_id = kwargs.pop("decoder_start_token_id", None)
+
+        # task specific arguments
+        self.task_specific_params = kwargs.pop("task_specific_params", None)
```

Comment:
Should it be `kwargs.pop("task_specific_params", {})` so we can directly test its keys?

Reply:
Guess it's the same pro/con discussion then above about `prefix = kwargs.pop("prefix", "")` or `kwargs.pop("prefix", "")` . If it's not None as a default it would at the moment be serialized and saved in all configs that don't have `task_specific_params`  or am I missing something?